cqrs-user-command-service/src/integrationTest/java/dev/amir/usercommand/UserCommandTest.java

```diff
@@ -90,7 +93,36 @@ void test_UpdateUserTest() throws Exception {
                         .contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk());
 
-        verify(userOutputPortMock).save(any(User.class));
+        verify(userOutputPortMock).update(any(User.class));
         verify(userMessageOutputPortMock).sendMessage(any(User.class));
     }
+
+    @Test
+    public void test_HandleUnknownExceptionForCreateUser() throws Exception {
+        when(userOutputPortMock.save(any(User.class))).thenThrow(InternalError.class);
+
+        mockMvc.perform(post("/user")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isInternalServerError())
+                .andExpect(content().string("Internal error"));
+    }
+
+    @Test
+    public void test_HandleUserNotFoundExceptionForUpdateUser() throws Exception {
+        UUID expectedUuid = UUID.randomUUID();
+        UserNotFoundException ex = new UserNotFoundException(expectedUuid);
+
+        when(userOutputPortMock.update(any(User.class))).thenThrow(new UserNotFoundException(expectedUuid));
+
+        File responseFile = ResourceUtils.getFile("classpath:responses/update-users-response.json");
+
+        mockMvc.perform(MockMvcRequestBuilders.put("/user/{userId}", expectedUuid)
+                        .content(Files.contentOf(responseFile, StandardCharsets.UTF_8))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound())
+                .andExpect(content().string("User not found"));
+
+        assertEquals("User with id: " + expectedUuid + " Not found", ex.getMessage());
```

Comment:
this assertion does not make sense, `ex` is a variable created in line 113. It is not a result value.
remove this line.